### PR TITLE
add: data/ennkay161.json

### PR DESCRIPTION
**Pull request by neerajkumar161**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5503"></a>

### CODE REVIEW

File: data/neerajkumar161.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+  "name": "Neeraj Kumar",
+  "type": "personal",
+  "bio": "Backend Developer | **Node.js** | TypeScript Lover",
+  "avatar": "",
```
Comment (priyankarpal) [inline data/neerajkumar161.json:5]: Currently, you don't need to add an ` avatar`. Delete this line.

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: data/neerajkumar161.json (hunk 1 above)
Comment (SaraJaoude) [reply data/neerajkumar161.json:5]:
```suggestion
```

**Review by SaraJaoude — APPROVED** [review]
looks good

**Comment by neerajkumar161** [comment]
Thank you for making changes and merging PR.

**Pull request merged**